Doc/library/tarfile.rst

```diff
@@ -1346,6 +1346,16 @@ parameter in :meth:`TarFile.add`::
     tar.add("foo", filter=reset)
     tar.close()
 
+How to create and write the archive to stdout using 
```

Comment:
The pre-commit hook is failing because of the trailing space ;)

```suggestion
How to create and write the archive to stdout using
```

Reply:
Ah, nice catch - I was wondering why it was failing CI